Terminal.Gui/Core/ConsoleDriver.cs

```diff
@@ -668,6 +792,23 @@ public abstract class ConsoleDriver {
 		// The format is rows, columns and 3 values on the last column: Rune, Attribute and Dirty Flag
 		internal abstract int [,,] Contents { get; }
 
+
+		/// <summary>
+		/// Determinates if the current console driver supports TrueColor output-
+		/// </summary>
+		public virtual bool SupportsTrueColorOutput { get => false; }
```

Comment:
I'm confused as to why there's ALSO a method in `CursesDriver.cs` called `CanColorTermTrueColor`. Why not just have `CursesDriver` implement an override of this method?

I think this method should be named `SupportsTrueColor` for tersness.

Reply:
This must be abstract in the ConsoleDriver.cs to force to be implemented in all drivers.

Reply:
The `CursesDriver` currently does not implement the TrueCollorAttribute feature.
On *unix, there is no *easy* way to determinate if the current terminal supports color output. 
Despite *ncruses* provides a check for color support, it tells only if the library is compiled with color output support.

The unofficial community way is to check the environment variable `COLORTERM` for some magic values. This does the static `CursesDriver.CanColorTermTrueColor` function. This variable is also set under WSL, so i just retrive this information in the `NetDriver`.

The virtual property `SupportsTrueColorOutput` tells if the used driver and the used terminal environment supports color output.

The property `UseTrueColor` is used to decide which mode should be used. If `SupportsTrueColorOutput` is set then `UseTrueColor` is also set, but can be uesed to enter baisc color rendering.